lib/agent.js

```diff
@@ -76,6 +76,17 @@ Agent.prototype.destroy = function () {
   if (this._transport) this._transport.destroy()
 }
 
+// These are metrics about the agent itself -- separate from the metrics
+// gathered on behalf of the using app and sent to APM server. Currently these
+// are only useful for internal debugging of the APM agent itself.
+//
+// **These stats are NOT a promised interface.**
+Agent.prototype._getStats = function () {
+  return {
+    apmclient: this._transport._getStats()
```

Comment:
Since `this._transport._getStats` is a new method on the transport/client, do we need to do a transport/client release and then be explicit about supporting that version of the client in the agent's package.json?  Or will that come later?

Reply:
Yah, that's fair. Perhaps less burdensome for coordinating releases for this small case, I could change this to check for the presence of `_transport._getStats()`. Then, at worst, `agent._getStats()` is an empty object.

Reply:
Done in https://github.com/elastic/apm-agent-nodejs/pull/2015/commits/8d1f810798699480584683bba80f85f1bccc5bf6

Seem reasonable?

Reply:
👍